mne_bids/tests/data/tiny_bids/sub-01/ses-eeg/eeg/sub-01_ses-eeg_task-rest_eeg.vmrk

```diff
@@ -1,13 +1,15 @@
 Brain Vision Data Exchange Marker File, Version 1.0
+; Exported using pybv 0.7.4
 
 [Common Infos]
 Codepage=UTF-8
 DataFile=sub-01_ses-eeg_task-rest_eeg.eeg
 
 [Marker Infos]
 ; Each entry: Mk<Marker number>=<Type>,<Description>,<Position in data points>,
-; <Size in data points>, <Channel number (0 = marker is related to all channels)>
+;             <Size in data points>, <Channel number (0 = marker is related to all channels)>
+;             <Date (YYYYMMDDhhmmssuuuuuu)>
 ; Fields are delimited by commas, some fields might be omitted (empty).
 ; Commas in type or description text are coded as "\1".
-Mk1=New Segment,,1,1,0,20000101120000000000
-Mk2=Comment,ControlBox is not connected via USB,1,1,0
+Mk1=Stimulus,S  1,1,1,0
+Mk2=Stimulus,S  2,1001,1,0
```

Comment:
This change seems to indicate that manual the events were never written in the previous version of the script, @sappelhoff 

Meaning we have a bug in MNE-BIDS, where the events passed to write_raw_bids() are not written in case the original data is just being copied over.

Here in this PR, where we enforce writing the data anew (because the raw is modified), the events show up.

@sappelhoff Can you follow what I mean?